search/src/Search.ts

```diff
@@ -0,0 +1,158 @@
+import { LitElement, css, html, unsafeCSS } from 'lit';
+import { unsafeHTML } from 'lit/directives/unsafe-html.js';
+import styles from './Search.scss';
```

Comment:
Vite build will add these styles to the `/style.css` when released, and mix it up with styles which come from other components
this is a problem, because `Search.scss` is needed for Shadow DOM of `dockit-search` component only and should never be added to global styles

I brought this up multiple times, but I can't convince Georges to change this behavior
not sure what else to do :) just waiting for more and more cases where it will become obvious we need to change the behavior or at least make it configurable

Reply:
Done and removed, following the convention on this repo